lib/drawSquareDoubles.c

```diff
@@ -14,6 +13,95 @@
 #define DEBUG2 '2'
 #define DEBUG3 '3'
 
+triangle* meshInit (int numberOfTris){
+    return (triangle*) malloc(numberOfTris * sizeof(triangle));
+}
```

Comment:
```suggestion
mesh meshInit(int numberOfTris)
{
    mesh newMesh;
    mesh.tris = (triangle*) malloc(numberOfTris * sizeof(triangle);
    mesh.numberOfTris = numberOfTris;
    
    return mesh
}```

Reply:
could make this func actually create an instance of a mesh struct rather than just its contents

Reply:
Yeah, this function did feel a bit singular.